### PR TITLE
Remove periods at the end of section descriptions

### LINKED ISSUE #38958: [Fix] Remove periods at the end of section descriptions
Some section descriptions in Pricing and Inventory end with periods that shouldn't be there. Let's remove them to boost consistency and avoid possible confusion.

image

image

### PR DESCRIPTION
**Pull request by octaedro**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR removes periods at the end of section descriptions.

The texts that have been modified are:
- `How to get started?.` to `How to get started?`.
- `Learn more about linked products.` to `Learn more about linked products`.

Closes #38958.

//cc @jarekmorawski 

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the New product editor and the `linked products` are enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Go to Product > Add New > Shipping
3. Press the interrogation mark next to the section's title `Fees & dimensions`.
4. Verify that the link in the tooltip says: `How to get started?` (without a period at the end).
5. Go to `Linked products` tab and verify that the tooltips next to the Upsells and Cross-sells sections say: `Learn more about linked products` (without a period at the end).

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php [hunk 1]
```diff
@@ -1153,8 +1153,8 @@ private function add_linked_products_group_blocks() {
 				'attributes' => array(
 					'title'       => __( 'Upsells', 'woocommerce' ),
 					'description' => sprintf(
-						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products.*/
-						__( 'Upsells are typically products that are extra profitable or better quality or more expensive. Experiment with combinations to boost sales. %1$sLearn more about linked products.%2$s', 'woocommerce' ),
+						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products link closing tag.*/
```
Comment (mattsherman) [inline plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php]: I like the clarification here... should update `%1$s` as well!

File: plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php [hunk 2]
```diff
@@ -1186,8 +1186,8 @@ private function add_linked_products_group_blocks() {
 				'attributes'     => array(
 					'title'       => __( 'Cross-sells', 'woocommerce' ),
 					'description' => sprintf(
-						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products.*/
-						__( 'By suggesting complementary products in the cart using cross-sells, you can significantly increase the average order value. %1$sLearn more about linked products.%2$s', 'woocommerce' ),
+						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products link closing tag.*/
```
Comment (mattsherman) [inline plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php]: I like the clarification here... should update `%1$s` as well!

**Review by mattsherman — CHANGES_REQUESTED** [review]
The translator comments need a little further updating.

**Review by mattsherman — COMMENTED** [review]
Tested well. Will approve once translator comments are updated!

File: plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php (hunk 1 above)
Comment (octaedro) [reply plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php]: Done in the commit [1983981](https://github.com/woocommerce/woocommerce/pull/46346/commits/1983981e9ff6bb1d7440f72a89a918f65e52ad46)

File: plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php (hunk 2 above)
Comment (octaedro) [reply plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php]: Done in the commit [1983981](https://github.com/woocommerce/woocommerce/pull/46346/commits/1983981e9ff6bb1d7440f72a89a918f65e52ad46)

**Comment by octaedro** [comment]
Thank you @mattsherman for reviewing this PR.
I just addressed the change you mentioned. Could you take another look at this PR?

**Review by mattsherman — APPROVED** [review]
Thanks for updating the translator comments. LGTM!

**Pull request merged**